README.md

```diff
@@ -1,33 +1,64 @@
+- Evaluation of the PR is limited to HVNC members (i.e., not for public discussion), although specific users may be asked for comments when warranted. If the PR is eventually merged, the discussion proposal is marked as ADOPTED.
+- If the PR is not merged for any reason (e.g., if we realize that the PR creates unforeseen issues with the existing nomenclature), the proposal will either be re-opened for more discussion or rejected at the discretion of the committee.
 
-1. Open the [new file](https://github.com/HGVSnomenclature/HVNC/new/main) dialogue.
-2. In the "Name your file..." field, use a `/` to create both a folder as well
-  as a document in one go. E.g., "repeats/proposal.md".
-3. Edit the document content, preferably using
-  [Markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).
-4. In the "Commit new file" section, give the draft proposal a name, e.g., "New proposal on repeats."
-5. Optionally add an extended description.
-6. Submit the new proposal by clicking the "Commit new file" button.
+## Committee Members
 
-### Edit proposal
+| Name | Term |
+|---|---|
+| [Johan den Dunnen](https://www.universiteitleiden.nl/en/staffmembers/johan-den-dunnen) (Nederland) | 2013 - |
+| [Marina DiStefano](https://www.linkedin.com/in/marina-distefano/) (United States) | 2022 - 2026 |
+ [Peter Freeman](https://research.manchester.ac.uk/en/persons/peter.j.freeman) (United Kingdom) | 2018 - |
+| [Reece Hart](https://linkedin.com/in/reece) (United States) | 2013 - |
+| [Ros Hastings](https://www.researchgate.net/profile/Ros-Hastings)† (United Kingdom) | 2021 - |
+| [Jeroen Laros](https://github.com/jfjlaros) (Nederland) | 2022 - 2026 |
+| [Terence Murphy]() (United States) | 2021 - 2026 |
+| [Rachel Taylor](https://www.linkedin.com/in/rachel-taylor-phd-90a57099/) (United Kingdom) | 2022 - 2026 |
+| [Alex Wagner](https://www.linkedin.com/in/alexphd/) (United States) | 2022 - 2026 |
 
-1. Navigate to the proposal folder and click on the proposal document name.
-2. Click on the pencil button to edit the document.
-3. In the "Commit changes" section, give a short description of the changes.
-4. Optionally add an extended description.
-5. Submit the changes by clicking the "Commit changes" button.
+† ISCN representative
 
+### Past Members
 
-## Discussions
+- 2013 - 2022
+  - Peter Taschner (Nederland)
+- 2014 - 2021
+  - Anne-Francoise Roux (France)
+- 2020 - 2021
+  - Oleg Agofonov (Norway)
+- 2013 - 2021
+  - Donna Maglott (United States)
+  - ISCN representative - Jean McGowan-Jordan (United States)
+  - HUGO representative - Stylianos Antonarakis (Switzerland)
+  - HGVS representative - Marc Greenblatt (United States)
+  - HVP / GV representative - Amy McAllister (United Kingdom)
+- 2013 - 2018
+  - HVP / GV representative - Tim Smith (Australia)
+- 2013 - 2014
+  - Els Dequeker (Belgium)
+
```

Comment:
Perhaps a section on the wider community who inputs. Ivo Fokkema for example. We may well get regular commitment from certain members of the community so it would be good to credit them.

Reply:
Great idea. Let's make that another revision so that we can wrap up this PR.